download.sh

```diff
@@ -4,12 +4,20 @@ set -e
 if [[ "$OSTYPE" == "linux-gnu" ]]; then
     echo 'downloading linux c-server-sdk shared library'
     rm -rf 'linux-gcc-64bit-dynamic.tar' 'include' 'lib'
-    curl -s -o 'linux-gcc-64bit-dynamic.tar' -L 'https://github.com/launchdarkly/c-server-sdk/releases/download/2.2.1/linux-gcc-64bit-dynamic.tar'
+    curl https://api.github.com/repos/launchdarkly/c-server-sdk/releases/latest | \
```

Comment:
Use the github to get the latest release. Then from that release find the linux or mac donwload URL. Then pipe that URL back into curl and download it.